libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -1022,31 +825,31 @@ void AC_AutoTune::load_tuned_gains()
         attitude_control->set_accel_pitch_max_cdss(0.0f);
     }
     if (roll_enabled()) {
-        if (!is_zero(tune_roll_rp)) {
+        if (!is_zero(tune_roll_rp) || allow_zero_rate_p()) {
             attitude_control->get_rate_roll_pid().kP(tune_roll_rp);
-            attitude_control->get_rate_roll_pid().kI(tune_roll_rp*AUTOTUNE_PI_RATIO_FINAL);
+            attitude_control->get_rate_roll_pid().kI(get_tuned_ri(axis));
             attitude_control->get_rate_roll_pid().kD(tune_roll_rd);
-            attitude_control->get_rate_roll_pid().ff(orig_roll_rff);
+            attitude_control->get_rate_roll_pid().ff(tune_roll_rff);
```

Comment:
I guess this is a small change in behaviour that we should check with @lthall.  Basically it means that if a multicopter users had set the ATC_RAT_RLL_FF to a non-zero value, after a successful (or unsuccessful?) autotune it would be set back to zero.

Reply:
Not necessarily. In the initialize gains method, I added a tune_roll_rff that is set to the same value as the orig_roll_rff gain.  So this should not change the FF gain after the vehicle is tuned.

Reply:
load and backup gain methods moved to subclasses